From jwboyer@linux.vnet.ibm.com  Mon Oct 27 11:43:29 2008
From: Josh Boyer <jwboyer@linux.vnet.ibm.com>
Date: Fri, 24 Oct 2008 10:10:42 -0400
Subject: Update stable tree documentation
To: greg@kroah.com
Cc: linux-kernel@vger.kernel.org
Message-ID: <20081024141042.GB2299@yoda.jdub.homelinux.org>
Content-Disposition: inline


Update the documentation for the stable tree rules to reflect
that device IDs and quirks are also suitable for -stable
kernels.

Signed-off-by: Josh Boyer <jwboyer@linux.vnet.ibm.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/stable_kernel_rules.txt |    1 +
 1 file changed, 1 insertion(+)

--- a/Documentation/stable_kernel_rules.txt
+++ b/Documentation/stable_kernel_rules.txt
@@ -12,6 +12,7 @@ Rules on what kind of patches are accept
    marked CONFIG_BROKEN), an oops, a hang, data corruption, a real
    security issue, or some "oh, that's not good" issue.  In short, something
    critical.
+ - New device IDs and quirks are also accepted.
  - No "theoretical race condition" issues, unless an explanation of how the
    race can be exploited is also provided.
  - It cannot contain any "trivial" fixes in it (spelling changes,
